Userland/Libraries/LibGUI/FileSystemModel.cpp

```diff
@@ -439,7 +440,7 @@ Variant FileSystemModel::data(const ModelIndex& index, ModelRole role) const
         case Column::Name:
             return node.name;
         case Column::Size:
-            return (int)node.size;
+            return AK::human_readable_size(node.size);
```

Comment:
please drop the AK namespace prefix here, this has a `using` directive:

```suggestion
            return human_readable_size(node.size);
```

Reply:
Thanks for the suggestion! I fixed it!